pkg/cloud/aws/services/ec2/ami.go

```diff
@@ -99,15 +97,14 @@ func (i images) Len() int {
 
 // Less reports whether the element with
 // index i should sort before the element with index j.
+// TODO(chuckha) Ignoring errors until this causes a problem
 func (i images) Less(k, j int) bool {
 	firstTime, err := time.Parse(createDateTimestampFormat, aws.StringValue(i[k].CreationDate))
 	if err != nil {
-		klog.Infof("unable to parse an AMI creation timestamp: %q", aws.StringValue(i[k].CreationDate))
 		return false
 	}
 	secondTime, err := time.Parse(createDateTimestampFormat, aws.StringValue(i[j].CreationDate))
 	if err != nil {
-		klog.Infof("unable to parse an AMI creation timestamp: %q", aws.StringValue(i[j].CreationDate))
```

Comment:
Removing these might be making it hard for users to debug, is there any way to bubble this up? Opening a related issue if there is no simple solution is completely ok

Reply:
Done. This is part of #716.